ui/src/containers/GphlWorkflowParametersDialog.jsx

```diff
@@ -251,13 +251,44 @@ function GphlWorkflowParametersDialog(props) {
                   } mb-5`}
                 >
                   <div className={`${styles.title} p-2`}>
-                    {ui_schema[rowKey]['ui:title'] ||
-                      schema.properties.indexing_solution?.title ||
-                      null}
+                    {rowKey === 'reffiles'
+                      ? schema.properties.reffiles?.title
+                      : rowKey === 'indexing_solution'
+                      ? schema.properties.indexing_solution?.title
+                      : ui_schema[rowKey]['ui:title']}
                   </div>
                   <Row>
-                    {ui_schema[rowKey]['ui:order'] ? (
-                      ui_schema[rowKey]['ui:order'].map((ColKey) => (
+                    {rowKey === 'indexing_solution' ? (
+                      ui_schema[rowKey]['ui:widget']?.includes('table') &&
+                      renderIndexingTable(
+                        ui_schema[rowKey][uiOptions],
+                        selected,
+                        onSelectRow,
+                      )
+                    ) : rowKey === 'reffiles' ? (
+                      // Specific textarea for "reffiles"
+                      schema.properties.reffiles.type === 'textarea' && (
```

Comment:
This textarea case is here  cause of it's future specificity 
it will have some regex for multi-line url validation in the future by @rhfogh